pkg/util/proxy.go

```diff
@@ -41,6 +41,15 @@ type ProxyConfig struct {
 func NewProxyConfig(httpProxy string, httpsProxy string, noProxy string) (*ProxyConfig, error) {
 	defaultScheme := "http"
 
+	// We only parse the Proxy Special char if it is coming from the config file
+	// or user defined flag. In case of Env variable we just ignore parsing.
```

Comment:
I think this need to be documented somewhere in https://docs.okd.io/latest/minishift/using/basic-usage.html#http-s-proxies